util/src/main/i18n/templates/moderation.properties

```diff
@@ -154,6 +154,9 @@ moderation.defuse.displayName = TNT Defuser
 # {1} = griefing player name
 moderation.defuse.message = {0} defused {1}'s TNT
 
+# {0} = defusing player name
+moderation.defuse.message.unknown = {0} defused a TNT
```

Comment:
Can players defuse TNT that doesn't belong to a teammate?

Reply:
I'm pretty certain they cannot. That message is for the case that the griefing player leaves and is accounted as null.

Reply:
Covering this review: https://github.com/PGMDev/PGM/pull/559#discussion_r453207526